deploy.rb

```diff
@@ -69,6 +69,7 @@ def run_command(command)
 build_commands = [
   "yarn run clean",
   "yarn install --pure-lockfile --cache-folder=.yarn-cache",
+  "spago build",
```

Comment:
And if we include the `spago build` in the apps themselves this should not be needed

Reply:
Rather than here